README.md

```diff
@@ -27,6 +27,14 @@ joining.
 
 You can build and test the Java code from a Unix shell.
 
+Prerequisites:
+1. Download Java version 11
+2. Check what version of Java Maven is using: ```mvn --version```
```

Comment:
Install Java Maven, with link?

Reply:
this is already in the readme in the Dependencies section

Reply:
Ah, then maybe we can combine the Dependencies + Prerequisite into one section?  later PR is okay.